config/go-app-ussd_popi_user_data.afr_ZA.json

```diff
@@ -209,19 +209,19 @@
     ],
     "Invalid phone number. Please re-enter (with no spaces)": [
         null,
-        ""
+        "Ongeldige selnommer. Tik asb weer in"
     ],
     "Sorry, the number you are trying to move to already has an active registration. To manage that registration, please redial from that number.": [
         null,
-        ""
+        "Jammer, die nommer wat jy nou wil gebruik het reeds 'n aktiewe registrasie. Om daardie registrasie te bestuur moet jy weer daardie nommer bel"
```

Comment:
This doesn't quite say the same thing as the english, the english says you should redial from that number and the afrikaans says you should redial that number.